frontend/src/components/player/PlayPage.jsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from "react-router-dom";
+import "../../static/css/play/playPage.css";
+
+export default function PlayPage() {
+    return (
+        <div className="page-container">
+            <div className="button-container">
+            <Link to={"join"}>
+            <button className="button">Unirse a partida</button>   
+            </Link>
```

Comment:
```suggestion
            <Link to={"join"}>
                <DButton className="button">Unirse a partida</button>   
            </Link>
```

La tabulación y DButton!